### PR TITLE
Update README.md

### LINKED ISSUE #2: [DOCS]: Improve the README.md file
Add the following to the README.md 

🔹A brief introduction to the project
🔹A table of contents for easy navigation
🔹Sections on features
🔹Installation
🔹Usage, contributing and licensing

### PR DESCRIPTION
**Pull request by mithindev**
closes #2 

Update README.md file with clearer project description, detailed instructions, and a table of contents to make it easier for users to understand and contribute to the project.

### CODE REVIEW

**Comment by Dun-sin** [comment]
Calm down my friend😭 didn't even assign you to anything.

**Comment by Dun-sin** [comment]
But thanks anyway

**Comment by mithindev** [comment]
I am sorry, But thanks.

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+3. Run `npm install` to install the project dependencies.
+4. Set up the database by running `npm run db:setup`.
+5. Start the server by running `npm start`.
+6. Navigate to `http://localhost:3000` in your web browser to use the application.
+
+Note: You will need to have Node.js and MongoDB installed on your machine to run this application.
+
+## 👨‍💻 Usage
+
+To use HearItFresh, simply navigate to the website and sign up for an account. Once logged in, you can search for songs, listen to them, and add them to playlists. You can also follow other users and see their music recommendations on your home page.
```
Comment (Dun-sin) [inline README.md:35]: This isn't what my project does

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+- Follow other users to see their music recommendations
+
+## 🛠️ Installation
+
+To install and run HearItFresh locally, follow these steps:
+
+1. Clone this repository to your local machine.
+2. Navigate to the project directory in your terminal.
+3. Run `npm install` to install the project dependencies.
+4. Set up the database by running `npm run db:setup`.
```
Comment (Dun-sin) [inline README.md:27]: there's no database

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+5. Start the server by running `npm start`.
+6. Navigate to `http://localhost:3000` in your web browser to use the application.
+
+Note: You will need to have Node.js and MongoDB installed on your machine to run this application.
```
Comment (Dun-sin) [inline README.md:31]: there's no backend either

File: README.md [hunk 4]
```diff
@@ -1 +1,55 @@
...
+
+HearItFresh is a web application that allows users to discover and share new music. Users can sign up for an account, search for and listen to songs, create and share playlists, and follow other users to see their music recommendations.
+
+## 💫 Features
+
+- User authentication and authorization
+- Search for songs by title, artist, or genre
+- Listen to songs directly in the browser
+- Create and share playlists with other users
+- Follow other users to see their music recommendations
```
Comment (Dun-sin) [inline README.md:18]: This isn't what my app does, did you use chatGPT to generate this readme lol

File: README.md [hunk 5]
```diff
@@ -1 +1,55 @@
+<div align="center">
+  
 ![image](https://user-images.githubusercontent.com/78784850/210045371-8f386335-88cf-4a65-9d00-6ac0e808269e.png)
+  
+</div>
+
+<h1 align="center">HearItFresh</h1>
+
+
+HearItFresh is a web application that allows users to discover and share new music. Users can sign up for an account, search for and listen to songs, create and share playlists, and follow other users to see their music recommendations.
```
Comment (Dun-sin) [inline README.md:10]: Once again, try to actually be genuine, rather than using ChatGPT to do this

**Review by Dun-sin — CHANGES_REQUESTED** [review]
70% of this PR is just wrong, if you still want to work on this properly let me know, otherwise i will close this

**Comment by mithindev** [comment]
Ok! I really want to work on this. Can you assign me this task?

**Comment by Dun-sin** [comment]
> Ok! I really want to work on this. Can you assign me this task?

I will assign you officially, Hopefully you see the review notes above

**Comment by mithindev** [comment]
Hello, @Dun-sin. Just want to remind you that I am not idle. I am working on it.

**Comment by Dun-sin** [comment]
> Hello, @Dun-sin. Just want to remind you that I am not idle. I am working on it.

well thanks for letting me know, i don't think it takes more than an hour tho, what's taking your time?

**Review by mithindev — COMMENTED** [review]
Viewed for correction.

**Comment by mithindev** [comment]
Hello @Dun-sin I have made the changes as you've guided. Should I make more changes? 
- Thank you so much for reviewing.

File: README.md [hunk 6]
```diff
@@ -1,51 +1,57 @@
 <div align="center">
-  
+
 ![image](https://user-images.githubusercontent.com/78784850/210045371-8f386335-88cf-4a65-9d00-6ac0e808269e.png)
-  
+
 </div>
 
 <h1 align="center">HearItFresh</h1>
+<h2 align="center">Explore fresh music inspired by the Spotify playlists you ❤️</h2>
```
Comment (Dun-sin) [inline README.md:8]:
```suggestion
<h2 align="center">Discover new tracks with Hear It Fresh! This user-friendly web app generates personalized Spotify playlists based on your favourite artists or a Spotify playlist link. Explore fresh music.</h2>
```

File: README.md [hunk 7]
```diff
@@ -1,51 +1,57 @@
...
-1. Clone this repository to your local machine.
-2. Navigate to the project directory in your terminal.
-3. Run `npm install` to install the project dependencies.
-4. Set up the database by running `npm run db:setup`.
-5. Start the server by running `npm start`.
-6. Navigate to `http://localhost:3000` in your web browser to use the application.
-
-Note: You will need to have Node.js and MongoDB installed on your machine to run this application.
+- Authentication of users is carried out through Spotify Login.
+- Create playlists using artist, track, or playlist seeds as a basis.
```
Comment (Dun-sin) [inline README.md:15]:
```suggestion
- Create new playlists using your favourite artists or an existing playlist
```

File: README.md [hunk 8]
```diff
@@ -1,51 +1,57 @@
...
+- Use a personal playlist as a basis to generate new playlists.
```
Comment (Dun-sin) [inline README.md:16]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,51 +1,57 @@
...
+- Tune parameters like different genre, none popular artists etc. to further customize the playlist
```
Comment (Dun-sin) [inline README.md:17]:
```suggestion
- Get better results by changing the parameters to either getting a playlist of different artists opposite to your taste or get less popular artists
```

**Review by Dun-sin — CHANGES_REQUESTED** [review]

**Comment by Dun-sin** [comment]
@mithindev it's much better, thanks for fixing them, just a little more change to Your PR, i gave the new review above

**Review by Dun-sin — APPROVED** [review]

**Comment by Dun-sin** [comment]
Thanks for this @mithindev

**Comment by mithindev** [comment]
Sure.  I will do it.

**Pull request merged**